docs/guides/02-content.md

```diff
@@ -0,0 +1,80 @@
+# Airship Content
+
+Airship is divided into three main packages:
+
+- @carto/airship-components: Web components ready to use in your application.
```

Comment:
Transform those items into a links to the npm packages

Reply:
Package descriptions should be consistent all around the docs. We should agree on the right one

Reply:
Yes, we need to give it a look and decide the official descriptions.